src/bluetooth.py

```diff
@@ -0,0 +1,30 @@
+import bluetooth
+import pexpect
+import time
+
+def connect(targetName):
+    targetAddress = None
+
+    while targetAddress == None:
+        nearby_devices = bluetooth.discover_devices()
+        print(len(nearby_devices))
+        for i in nearby_devices:
+            print(bluetooth.lookup_name(i))
+            if targetName == bluetooth.lookup_name(i):
+                targetAddress = i
+
+        time.sleep(1)
+
+    print(targetAddress)
+
+
+    child = pexpect.spawn('bluetoothctl')
+    child.sendline('connect ' + targetAddress)
+    child.sendline('exit')
+    child.interact()
+
+
+if __name__ == '__main__':
+    targetName = "Xbox Wireless Controller"
```

Comment:
There can be plenty of devices with the name "Xbox Wireless Controller." It's a lot more reliable and precise to use the Bluetooth address.

Reply:
Also you could just pass this as a parameter. Probably don't need a variable for it.

Reply:
I agree, I can take that out once I know the address for our controller, but this just made it easier to test at home with my controller.